devfiles/no-custom-image/README.md

```diff
@@ -0,0 +1,38 @@
+# no-custom-image
+
+The provided devfile allows you to start developing your maven built applications without the need of a custom image.
+
+## Prerequisites
+
+- OpenShift Do (ODO) CLI.
+
+- Kubernetes Cluster.
+
+
+## About The Devfile
+
+- It uses a Volume mounted to /home/user/.m2 for runtime caching.
+
+- No outer loop deployment entries.
+
+
+## Inner loop development
+
+1. Copy the devfile to your application directory.
+
+```
+$ ls /myproject/myApplication
```

Comment:
Instead of "Copy the devfile" you could say:
> from your project root:    `curl -L  <url> -o devfile.yaml` 

I see what you're trying to say but a bit clearer when the main action is what you type.

Reply:
Changed this along with others.